core/chains/evm/gas/chain_specific.go

```diff
@@ -9,9 +9,9 @@ import (
 // chainSpecificIsUsable allows for additional logic specific to a particular
 // Config that determines whether a transaction should be used for gas estimation
 func chainSpecificIsUsable(tx evmtypes.Transaction, baseFee *assets.Wei, chainType config.ChainType, minGasPriceWei *assets.Wei) bool {
-	if chainType == config.ChainGnosis || chainType == config.ChainXDai {
+	if chainType == config.ChainGnosis || chainType == config.ChainXDai || chainType == config.ChainXLayer {
 		// GasPrice 0 on most chains is great since it indicates cheap/free transactions.
```

Comment:
cc @samsondav as you seem to have implemented this back then?

Reply:
I couldn't find any resources on this for Gnosis, and there don't seem to be any gasless transactions anymore happening on Gnosis. Do you remember what info you based this on?